File: operation-spring/core/dto/DashboardResetDto.java
package com.prompt.operation.core.dto;

import com.prompt.operation.entity.dto.*;

import java.util.List;

public class DashboardResetDto {

    private List<CostElementOperation> costElementOperations;
    private List<ExpectedParcelOperation> expectedParcelOperations;
    private List<PaymentOperation> paymentOperations;
    private List<RealizedParcelOperation> realizedParcelOperations;
    private List<TechnicalAreaOperation> technicalAreaOperations;

    public DashboardResetDto(
        List<CostElementOperation> costElementOperations,
        List<ExpectedParcelOperation> expectedParcelOperations,
        List<PaymentOperation> paymentOperations,
        List<RealizedParcelOperation> realizedParcelOperations,
        List<TechnicalAreaOperation> technicalAreaOperations
    ) {
        this.costElementOperations = costElementOperations;
        this.expectedParcelOperations = expectedParcelOperations;
        this.paymentOperations = paymentOperations;
        this.realizedParcelOperations = realizedParcelOperations;
        this.technicalAreaOperations = technicalAreaOperations;
    }

    public DashboardResetDto() {
    }

    public List<CostElementOperation> getCostElementOperations() {
        return costElementOperations;
    }

    public void setCostElementOperations(List<CostElementOperation> costElementOperations) {
        this.costElementOperations = costElementOperations;
    }

    public List<ExpectedParcelOperation> getExpectedParcelOperations() {
        return expectedParcelOperations;
    }

    public void setExpectedParcelOperations(List<ExpectedParcelOperation> expectedParcelOperations) {
        this.expectedParcelOperations = expectedParcelOperations;
    }

    public List<PaymentOperation> getPaymentOperations() {
        return paymentOperations;
    }

    public void setPaymentOperations(List<PaymentOperation> paymentOperations) {
        this.paymentOperations = paymentOperations;
    }

    public List<RealizedParcelOperation> getRealizedParcelOperations() {
        return realizedParcelOperations;
    }

    public void setRealizedParcelOperations(List<RealizedParcelOperation> realizedParcelOperations) {
        this.realizedParcelOperations = realizedParcelOperations;
    }

    public List<TechnicalAreaOperation> getTechnicalAreaOperations() {
        return technicalAreaOperations;
    }

    public void setTechnicalAreaOperations(List<TechnicalAreaOperation> technicalAreaOperations) {
        this.technicalAreaOperations = technicalAreaOperations;
    }
}

File: operation-nest/core/domain/dashboard-reset.dto.ts
import { CostElementOperation } from '../../entity/cost-element-operation/cost-element-operation';
import { ExpectedParcelOperation } from '../../entity/expected-parcel-operation/expected-parcel-operation';
import { PaymentOperation } from '../../entity/payment-operation/payment-operation';
import { RealizedParcelOperation } from '../../entity/realized-parcel-operation/realized-parcel-operation';
import { TechnicalAreaOperation } from '../../entity/technical-area-operation/technical-area-operation';

export class DashboardResetDto {

    costElementOperations: CostElementOperation[];
    expectedParcelOperations: ExpectedParcelOperation[];
    paymentOperations: PaymentOperation[];
    realizedParcelOperations: RealizedParcelOperation[];
    technicalAreaOperations: TechnicalAreaOperation[];

    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
    constructor(data?: any) {
        this.costElementOperations = data && data.costElementOperations || [];
        this.expectedParcelOperations = data && data.expectedParcelOperations || [];
        this.paymentOperations = data && data.paymentOperations || [];
        this.realizedParcelOperations = data && data.realizedParcelOperations || [];
        this.technicalAreaOperations = data && data.technicalAreaOperations || [];
    }

}
